scripts/packageTorClient.js

```diff
@@ -44,7 +44,7 @@ const downloadTorClient = (platform) => {
   mkdirp.sync(torPath)
 
   const torClient = path.join(torPath, torFilename)
-  const cmd = 'curl -o ' + torClient + ' ' + torURL
+  const cmd = 'aws s3 cp ' + torURL + ' ' + torClient
```

Comment:
does this work?

Reply:
yes, it does https://ci.brave.com/view/all-extensions/job/brave-core-ext-tor-client-update-publish-dev/23/